libnavui-androidauto/build.gradle

```diff
@@ -40,13 +40,12 @@ dependencies {
 
     // This defines the minimum version of Navigation which is included in this SDK. To upgrade the
     // Navigation versions, you can specify a newer version in your downstream build.gradle.
-    def carNavVersion = "2.9.0-alpha.3"
-    api("com.mapbox.navigation:android:${carNavVersion}")
+    releaseApi("com.mapbox.navigation:android:2.9.0-alpha.3")
```

Comment:
https://github.com/mapbox/mapbox-navigation-android/releases/tag/v2.9.0-alpha.4

^ Still requires `10.8.0`

Reply:
Updated, but we have to keep a separate dependency for debug.